app/about/intro/route.js

```diff
@@ -4,6 +4,10 @@ import ChannelsIndexRoute from 'radio4000/channels/index/route'
 
 export default ChannelsIndexRoute.extend({
```

Comment:
Here either:
- just a normal route
- or do not rewrite the model hook

Reply:
I vote 2, so it's not just "latest featured" but random like on the homepage. But both work of course :]